src/carousel.scss

```diff
@@ -172,7 +185,13 @@ $button: #{$fd-namespace}-button;
 
   &--no-navigation {
     .#{$block}__button.#{$button} {
-      display: none;
+      display: none !important;
```

Comment:
Is important really necessary here? can we use specificity instead of `!important` here?

You can see this article why using `!important` is bad practise https://uxengineer.com/css-specificity-avoid-important-css/